docs/cvl/methods.md

```diff
@@ -211,8 +327,14 @@ itself).  They differ in the assumptions made about the return value:
    returned values is *not* assumed to match the requested number, unless
    {ref}`-optimisticReturnsize` is specified.
 
+```{warning}
+Using `CONSTANT` and `PER_CALLEE_CONSTANT` summaries for functions that have
```

Comment:
Really? Do we have an example?

Reply:
I took this from the warning in the source code (see `Summarizer.generateConstSummToInline`)

Reply:
worth testing at some later point